_commands/tools/diff.md

```diff
@@ -0,0 +1,77 @@
+---
+---
+
+diff
+-------
+
+`diff` is a tool used to compare files line by line.
+
+~~~ bash
+$ diff [option]... [file1] [file2] 
+~~~
+
+<!--more-->
+
+### Useful Options / Examples
+
+#### Example command
+
+#### `diff -y -W 80 -q test1.txt test2.txt`
+~~~ bash
+$ diff -y test1.txt test2.txt
+this is line one of a file           this is line one of a file
+this line differs                  | this line is different
```

Comment:
You can't see the 'break it down' without scrolling back and forth, could you move explanations to immediately under the command they explain? Or at least add a little blurb?